main.go

```diff
@@ -353,11 +352,8 @@ func setup(ctx context.Context, deps *setupDeps) bool {
 	env := os.Environ()
 	sort.Strings(env)
 	if err = deps.server.UpdateSettingsFromEnv(env); err != nil {
```

Comment:
Maybe the whole UpdateSettingsFromEnv or most of it now belongs to the `models`.

Reply:
I don't think that UpdateSettingsFromEnv should be in models. models shouldn't know anything about env variables.